src/OpenTelemetry.Extensions/CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
   can still be used with .NET5.0 apps.
   ([#617](https://github.com/open-telemetry/opentelemetry-dotnet/pull/617))
 
+## 1.0.0-beta.4
```

Comment:
I think that you should take care about changelog once more time. It should be part of unrelesed or all from unreleaset should be here.

Reply:
fixed

Reply:
@jvmlet You have updated the CHANGELOG to add an entry for `1.4.0-beta.4`. Did you want to have the `1.4.0-beta.4` version of this package released?

Reply:
Yes  please